src/Core/View.js

```diff
@@ -297,6 +298,7 @@ class View extends THREE.EventDispatcher {
         }
 
         window.removeEventListener('resize', this._resizeListener);
+        this.dispatchEvent({ type: VIEW_EVENTS.REMOVED });
```

Comment:
Can you dispatch it at the end of this method instead please?

Reply:
I wanted to but we have an event cleaner, so we need to do it before it

```js
 // remove alls events
this.removeAllEvents();

```

Reply:
Oh ok, I missed that. This is not ideal since if a user registers to this event, he would expect the view and everything related to the view to be disposed which won't be the case. What about moving `this.removeAllEvents()` at the end of the method too ?